sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/TracerProvider.java

```diff
@@ -33,17 +33,18 @@ public boolean isEnabled() {
      * new span will be added as a child, otherwise the span will be created and added to the context and any downstream
      * start calls will use the created span as the parent.
      *
+     * @param serviceBaseName the service name to be appended to the span name.
      * @param context Additional metadata that is passed through the call stack.
      * @param processKind the invoking process type.
      * @return An updated context object.
      */
-    public Context startSpan(Context context, ProcessKind processKind) {
+    public Context startSpan(String serviceBaseName, Context context, ProcessKind processKind) {
```

Comment:
Does this break Event Hubs?

Reply:
I updated Eventhubs too in this PR. It should be implementation details.